aten/src/ATen/native/cuda/ReduceSumProdKernel.cu

```diff
@@ -57,6 +57,30 @@ struct nansum_functor {
   }
 };
 
+const char nansum_name[] = "nansum";
```

Comment:
```suggestion
constexpr char nansum_name[] = "nansum";
```

Reply:
Sure. Updated it.

I think all the Jiterator ops are currently using `const` instead of `constexpr`. I'll send another PR updating it :)
Although I'm not sure why we need `constexpr` here 😅 So reading this great post from [stack-overflow](https://stackoverflow.com/questions/14116003/whats-the-difference-between-constexpr-and-const).